api/v1alpha1/common_types.go

```diff
@@ -21,6 +21,8 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+const PauseAnnotationKey = "admission-webhook.pingcap.com/pause"
```

Comment:
This feature isn't related to `admission-webhook`, I think we can use the other key, eg: "experiment.pingcap.com/pause` or others.

Reply:
Make sense.